mapApp/static/mapApp/js/recentReports.js

```diff
@@ -31,10 +31,12 @@ function addData(data){
   nearmissesLayer = geojsonCircleMarker( data.nearmisses , "nearmiss").addTo(map);
   hazardsLayer = geojsonCircleMarker( data.hazards , "hazard").addTo(map);
   theftsLayer = geojsonCircleMarker( data.thefts , "theft").addTo(map);
-  rois = geojsonPolygonMarker(data.rois).addTo(map);
-
+  rois = geojsonPolygonMarker(data.rois);
+  rois.addTo(map);
   // Fit map extent to alert areas boundary
-  map.fitBounds(rois);
+  if (rois.getBounds().isValid()){
```

Comment:
Same idea here, chained function calls always throw a red flag for me when I don't have a good understanding of the code!